pkg/install/stack/common/validation.go

```diff
@@ -140,3 +141,18 @@ func randomString() (string, error) {
 
 	return base64.RawURLEncoding.EncodeToString(b), nil
 }
+
+// ValidatePublicIp validates whether ip provided is public
+func ValidatePublicIp(ipAddress string) bool {
```

Comment:
Boolean functions should just be called "Is...". And this doesn't need to be anexported function.